apis/python/src/tiledbsoma/_dataframe.py

```diff
@@ -436,6 +436,8 @@ def write(
                         else:
                             extend_vals = np.array(update_vals, enmr.dtype)
                         new_enmr = enmr.extend(extend_vals)
+                        df = pd.Categorical(col.chunk(0).to_pandas(), new_enmr.values())
```

Comment:
This new code points out another potential bug -- what happens if the column's ChunkedArray has more than one chunk, where each chunk has a different set dictionary (enum) values?

Here is a real example I pulled from a multi-chunk column:
```
[<pyarrow.lib.ChunkedArray object at 0x7fd593a13ec0>
 [
 
   -- dictionary:
     [
       "A",
       "B"
     ]
   -- indices:
     [
       0,
       1,
       0
     ],
 
   -- dictionary:
     [
       "B",
       "C"
     ]
   -- indices:
     [
       0,
       1,
       0
     ]
 ]]```

As I read this code, it will only handle the information in chunk `0`, which is definitely incorrect....

Reply:
Actually I see more examples of the same bug in subsequent code (e.g., anywhere `col.chunk(0)` is hard-wired.